### DISCUSSION TITLE
Membership Status

### DISCUSSION THREAD

**Discussion by peterluu** [ChurchApps/ChurchAppsSupport#337] [Chums] [answered]
Hi team, I'd love to either be able to customize the Membership Status field. Or ask for some more options. We differentiate between members and regular attendees (who are more than visitors), so we'd like to be able to indicate that in the database (and as a result, build reports based on this data).

**Comment by peterluu** [comment]
In addition to this, I'd like the ability to mark whether a person is still active or inactive at church. We're required to keep an up to date roll of active membership. At the same time, I'd like to be able to maintain legacy data of previous members. It would also be nice to be able to indicate the reason for their inactivity; such as, moved location, new church or deceased.

**Comment by jzongker** [comment, marked as answer]
Good suggestions.  I think adding "Regular Attendee" makes sense on the membership status.  For the inactivity reason, I think either the Notes field could be used or it could be added as a custom field (added via Forms section).

It sounds like we still need one more membership status.  "Inactive"?  So membership options would be "Visitor", "Regular Attendee", "Member", "Staff", or "Inactive"...

**Comment by jzongker** [comment]
Added **Regular Attendee** and **Inactive** as new membership statuses.
